subsys/net/lib/lwm2m_client_utils/lwm2m/ecid_signal_meas_info.c

```diff
@@ -104,6 +104,11 @@ static struct lwm2m_engine_obj_inst *signal_meas_info_create(uint16_t obj_inst_i
 		}
```

Comment:
Indentation of these braces could also be fixed.

Reply:
I don't spot any problem.